main.go

```diff
@@ -31,14 +42,23 @@ func main() {
 	}
 
 	log.Println("Removing Service Catalog release")
-	cleaner.RemoveRelease(ServiceCatalogReleaseName)
+	err = cleaner.RemoveRelease(ServiceCatalogReleaseName)
+	if err != nil {
+		panic(err)
+	}
 
 	log.Println("Removing service-catalog-addons release")
 	cleaner.RemoveRelease(ServiceCatalogAddonsReleaseName)
+	if err != nil {
+		panic(err)
+	}
 
 	log.Println("Removing Helm Broker release")
 	cleaner.RemoveRelease(HelmBrokerReleaseName)
-	time.Sleep(2 * time.Second)
+	if err != nil {
+		panic(err)
+	}
+	time.Sleep(10 * time.Second)
```

Comment:
In my tests removing resources some times takes quite a while.
Should we maybe double check whether the resource is actually removed, before we proceed? This kind of sanity check could also be done in RemoveRelease().